homeassistant/components/influxdb/sensor.py

```diff
@@ -48,6 +49,7 @@
     DEFAULT_RANGE_STOP,
     INFLUX_CONF_VALUE,
     INFLUX_CONF_VALUE_V2,
+    LANGUAGE_INFLUXQL_RAW,
```

Comment:
Why is this `INFLUXQL_RAW`? Shouldn't you also be able to specify a flux query in the raw as well? It's raw so its just text either way.

Reply:
Changed to `LANGUAGE_RAW` in the e7dfb9f commit. Used to define the "raw" type of configuration that can be used in V1 or V2 mode